src/api/roles/shared/chat.py

```
@@ -0,0 +1,83 @@
+    chat = Chat(client_coach_relationship_id=payload.relationship_id)
+    db.add(chat)
+    db.flush()
+    db.commit()
+
+    if chat.id is None:
+        raise HTTPException(500, detail="Chat creation failed")
+    
+    return NewChatResponse(chat_id=chat.id)
+
+@router.post("/send_message/{chat_id}", response_model=SendMessageResponse)
+def send_message(chat_id: int, message_text: str, db = Depends(get_session), acc: Account = Depends(get_client_account)):
```

Ruff (0.15.10)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)